### PR TITLE
JRE min version

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java [hunk 1]
```diff
@@ -141,6 +141,9 @@
   @Parameter(property = "sonar.addMavenDescriptor", defaultValue = "true")
   private boolean addMavenDescriptor;
 
+  @Parameter(defaultValue = "1.8", required = true)
```
Comment (jblievremont) [inline src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java:144]: What is the expected format for this attribute, is it supposed to be based on the Java version syntax (jumping from 1.8 to 9, 10, 11, ...) or can we use a more straightforward one?

Comment (dbmeneses) [reply src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java:144]: Possibly a crazy idea: can it be derived from `maven.compiler.target` by default to ensure that there is no inconsistency?

Comment (henryju) [reply src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java:144]: @jblievremont my plan is to rely on system property `java.specification.version` at runtime to check is the requirement is satisfied. So we will indeed have to manage the version syntax change. But to me this is not a big deal, AFAIR this is already done in SLVSCODE, parsing it as a float, and using natural comparison (1.8 < 9 < 10 < 11 <...)

Comment (henryju) [reply src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java:144]: @dbmeneses I though about it: we would have to do something similar to the Scanner for Maven (introspect the maven-compiler-plugin), and possibly fix https://jira.sonarsource.com/browse/MSONAR-174. Not sure it is worth the effort...

**Pull request merged**